### PR TITLE
WIP: Issue/6/1

### PR DESCRIPTION
**Pull request by marimeireles**

### CODE REVIEW

File: src/widget.ts [hunk 1]
```diff
@@ -162,13 +163,10 @@ class CytoscapeView extends DOMWidgetView {
   init_render() {
     this.is_rendered = true;
     console.log('🦋')
-    console.log(this.model.get('graph')["attributes"]['nodes'])
     this.cytoscape_obj = cytoscape({
       container: this.el,
       elements: this.model.get('graph').converts_dict(),
```
Comment (marimeireles) [inline src/widget.ts]: I have a question here. Every time I pause this and inspect `this.model.get('graph')` on my debugger I get an empty dictionary, it's not the complex object I expected aka the class `GraphModel` with its methods, so I can't call `converts_dict()` here... But I don't really understand why.
From my understanding I used the [serialization](https://ipywidgets.readthedocs.io/en/stable/examples/Widget%20Low%20Level.html#Serialization-of-widget-attributes) stuff the right way.

Comment (marimeireles) [reply src/widget.ts]: My error:
```
widget.ts:169 Uncaught (in promise) TypeError: Cannot read property 'converts_dict' of undefined
    at p.init_render (widget.ts:169)
    at widget.ts:153
```

File: src/widget.ts [hunk 2]
```diff
@@ -28,6 +32,86 @@ cytoscape.use( popper );
...
+      _model_module_version: GraphModel.model_module_version,
+      nodes: [],
+      edges: [],
+    }
+  };
+
+  static serializers: ISerializers = {
+      nodes: { deserialize: widgets.unpack_models },
+      edges: { deserialize: widgets.unpack_models },
+      ...WidgetModel.serializers
```
Comment (wolfv) [inline src/widget.ts]: I think you need to add `graph: { deserialize: widgets.unpack_models }` here

Comment (marimeireles) [reply src/widget.ts]: Thanks Wolf! 🐺
This throws a different error... That doesn't make a lot of sense for me either
```
Uncaught (in promise) TypeError: this.model.get(...).converts_dict is not a function
    at p.init_render (widget.ts:169)
    at widget.ts:154
```
[Might](https://stackoverflow.com/questions/46943255/angular-resolve-error-error-uncaught-in-promise-typeerror-resolver-is-not-a) be a problem with this `Guard` concept? But I tried to pass `deserialize: widgets.unpack_models` in different forms and wasn't successful either.

Comment (wolfv) [reply src/widget.ts]: can you check what class the result of `this.model.get('graph')` is?

I think it will _not_ be the type you expect.

Comment (marimeireles) [reply src/widget.ts]: It's an object now, but still doesn't have the method `convers_dict`.
Screen Shot 2020-03-17 at 1 18 35 PM

**Pull request closed without merging**